scripts/test.sh

```diff
@@ -85,6 +87,15 @@ run_make() {
     CTEST_OUTPUT_ON_FAILURE=1 CTEST_PARALLEL_LEVEL="${TEST_PARALLEL_LEVEL}" make "-j${TEST_PARALLEL_LEVEL}" "$@"
 }
 
+#Make doxygen
+if [[$BUILD_DOCS = 1]]; then
```

Comment:
It does not work because there is no space between `[[` and `$`. (don't know why CI doesn't shout louder though...). Also, it's before the CMake step which shouldn't work.

Please try to follow the same pattern as the other stages: `echo ">> xxx"`, the `$TEST_DRYRUN` check, using `run_make`...
I would also put the docs building at the very end  to follow the general practice that tests more likely to fail should precede others. As it is not a test itself and should not fail, we might as well do it at the end.

Reply:
Thanks. Good advise! I'll try it later.